.github/workflows/changesets.yml

```diff
@@ -21,19 +18,30 @@ jobs:
               with:
                   fetch-depth: 0
 
+            - name: setup pnpm
+              uses: pnpm/action-setup@v4
+
             - name: setup node.js
               uses: actions/setup-node@v3
               with:
                   node-version: 20
```

Comment:
you can also add a line here `cache: pnpm` which will use a pre-installed version of pnpm to save you from having to download and install it